airflow/providers/amazon/aws/operators/redshift_cluster.py

```diff
@@ -423,6 +424,7 @@ class RedshiftResumeClusterOperator(BaseOperator):
         The default connection id is ``aws_default``
     :param deferrable: Run operator in deferrable mode
     :param poll_interval: Time (in seconds) to wait between two consecutive calls to check cluster state
+    :param deferrable: If True, the operator will run as a deferrable operator.
```

Comment:
There already is a param for deferrable above?

Reply:
Yep, that came from the rebase. I'm removing one of them. Thanks!